src/compiler/crystal/command.cr

```diff
@@ -411,7 +411,11 @@ class Crystal::Command
           compiler.verbose = true
         end
         opts.on("--static", "Link statically") do
-          compiler.static = true
+          {% if flag?(:darwin) %}
```

Comment:
I think we can add source link here:

```cr
          # https://developer.apple.com/library/content/qa/qa1118/_index.html
          {% if flag?(:darwin) %}
            raise "macOS doesn't support static linking"
          {% else %}
            compiler.static = true
          {% end %}
```

Reply:
Perhaps even include the link in the error output?